scripts/server

```diff
@@ -6,29 +6,115 @@ if [[ -n "${ASHLAR_BLUEPRINT_DEBUG}" ]]; then
     set -x
 fi
 
+# Import database methods from the directory path of this script
+CURDIR="${BASH_SOURCE%/*}"
+. "${CURDIR}/_init_db.sh"
+
 function usage() {
-    echo -n "Usage: $(basename "$0")
+    echo -n "Usage: $(basename "$0") {ashlar,editor} [CMD]
+
+Starts servers using docker-compose. If an argument is specified, [CMD]
+will pass the command string into the corresponding service.
 
-Starts servers using docker-compose.
+Argument options:
+    - ashlar: Starts the Ashlar server instance on port 8000.
+    - editor: Start the schema editor on port 9000.
+    - <none>: Start all containers.
 "
 }
 
+function checkport() {
+    #
+    # Check that a given port has no processes running on it.
+    #
+    if [[ $(netstat -nlp | grep ":::${1}") ]];
+    then
+        echo "It appears you already have a process running on port ${1}:"
+        echo
+        echo $(netstat -nlp | grep ":::${1}")
+        echo
+        echo "Terminate that process, or alter docker-compose.yml to use another port."
+        exit 1
+    fi
+}
+
+function run_ashlar_service() {
+    #
+    # Run the Ashlar server instance.
+    #
+    checkport 8000
+    init_db
+    if [ $# -eq 0 ];
+    then
+        # No arguments -- default to running a Django dev server
+        docker-compose run --rm --service-ports ashlar manage.py runserver 0.0.0.0:8000
+    else
+        # User passed in arguments -- run their command
+        docker-compose run --rm --service-ports ashlar "$@"
+    fi
+}
+
+function run_editor_service() {
+    #
+    # Run the schema editor app.
+    #
+    checkport 8000
+    init_db
+    docker-compose up -d ashlar
```

Comment:
Also, as we discussed, it might be good to try out the Azavea PostGIS image to see if it'll be faster to boot up and also improve the data persistence issues we were seeing. It might be possible to switch back to vanilla `docker-compose up` and strip out some of this DB-initialization logic entirely if the image works better.

Reply:
The Azavea PostGIS image loads much faster! I'm going to keep the database-initialization logic for now, since initial testing indicates that it can take between 1-3 seconds longer to load the DB than the app on my machine. But still, that's nearly 10 seconds faster, which is going to be much less frustrating for development.